moped-editor/src/components/ExternalLink.js

```diff
@@ -1,8 +1,20 @@
 import React from "react";
 import Link from "@material-ui/core/Link";
+import { makeStyles } from "@material-ui/core";
 import OpenInNewIcon from "@material-ui/icons/OpenInNew";
 
-const ExternalLink = ({ url, text, linkColor, underline }) => {
+const useStyles = makeStyles(theme => ({
+  iconStyle: {
+    fontSize: "14px",
+    marginLeft: "2px",
+    position: "relative",
+    bottom: "-3px",
+  },
```

Comment:
Aesthetic opinion / question - not my forte.

I think this matches the font better since it is between the baseline and the capline. Are we losing anything by not having it emulating a _superscript_ style? I like how this looks TBH; just curious.

Reply:
The way it was raised above the font was endlessly annoying to me. I asked Amenity if it was ok to change the positioning to get a gut check if it was just me and she seemed to have a preference for a more aligned icon. So I went for it.

Reply:
I am totally on board with it - I think it's 👍 💯. Thanks for making it feel like a normal glyph again!